try:
    raise ValueError
except ValueError:
    print('err')
